src/main/java/org/isf/stat/manager/JasperReportsManager.java

```diff
@@ -437,93 +436,106 @@ public JasperReportResultDto getGenericReportPharmaceuticalOrderPdf(String jaspe
                     MessageBundle.getMessage("angal.stat.reporterror.msg"), OHSeverityLevel.ERROR));
         }
     }
```

Comment:
there are some `catch(` (without space) in this class

Reply:
Fixed